Updated third-party apps sections

tamigoodall opened the pull request:
Updated links from other files
Consolidate files
Deleted 3 files
Added new guidance from SME

Review links:

- https://docs-content-staging-pr-8573.herokuapp.com/docs/applications
- https://docs-content-staging-pr-8573.herokuapp.com/docs/applications/concepts/app-types-first-third-party

<!---
Notes:
- Your PR should conform to our [Contributing Guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md)
- If applicable, add details to the [update feed](https://github.com/auth0/docs/tree/master/updates)
- Make sure your PR gets deployed to a Heroku [Review App](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#review-apps) without errors
- Please include a short description
- If your PR is a work in progress, title it [DO NOT MERGE]
--->

tamigoodall committed:
Updated third-party apps sections

Updated links from other files
Consolidate files
Deleted 3 files
Added new guidance from SME

tamigoodall committed:
Minor changes

Deleted redundant docs connections.md
Minor changes to applications/index.md
Added link to monitor-applications.md
Updated redirects.js

tamigoodall committed:
Update index.md

Added set up links

tamigoodall committed:
Minor changes

Minor changes

cocojoe commented:
Hey @tamigoodall what do you need to be reviewed from SDKs, I had a quick look but the content but most seems dashboard related. Thanks

tamigoodall committed:
Update index.md

topic links include nesting

tamigoodall committed:
Simplified main apps index.md page

Simplified main apps index.md page
Minor changes to register... descriptions and titles for consistency

tamigoodall committed:
Minor changes

Minor changes

tamigoodall commented:
> Hey @tamigoodall what do you need to be reviewed from SDKs, I had a quick look but the content but most seems dashboard related. Thanks

Hi @cocojoe That review request was sent by mistake. Not sure how it was activated. Sorry ffor wasting your time.

tamigoodall committed:
Minor fixes

Minor fixes

tamigoodall committed:
Deleted redundant file and updated lines

Deleted redundant file and updated lines

tamigoodall committed:
Typo fix and moved paragraph

Typo fix and moved paragraph

tamigoodall committed:
Update app-types-first-third-party.md

typo

jeffreylees commented on articles/api-auth/apis.md:
```diff
@@ -62,7 +62,7 @@ The other available views for your API are:
 - **Scopes**: here you can define the scopes for this API, by setting a name and a description.
 
-- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. By default, this grant is **enabled* for [Regular Web Applications and Machine to Machine Applications](/applications/concepts/app-types-auth0). You can authorize any of these applications to request Access Tokens for your API. Optionally, you can select a subset of the defined ...
+- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. By default, this grant is **enabled* for [Regular Web Applications and Machine to Machine Applications](/applications). You can authorize any of these applications to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to  limit your au ...
```
```suggestion
- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. By default, this grant is **enabled** for [Regular Web Applications and Machine to Machine Applications](/applications). You can authorize any of these applications to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to limit your authorized application's access. 
```

Commit:
Typo fixes for articles/api-auth/apis.md

jeffreylees commented on articles/api-auth/tutorials/client-credentials/customize-with-hooks.md:
```diff
@@ -15,15 +15,11 @@ useCase:
-Hooks allow you to customize the behavior of Auth0 using Node.js code.
-
-They are actually [Webtasks](https://webtask.io/), associated with specific extensibility points of the Auth0 platform (like the Client Credentials grant). Auth0 invokes the Hooks at runtime to execute your custom logic.
+Hooks allow you to customize the behavior of Auth0 using Node.js code.They are actually [Webtasks](https://webtask.io/), associated with specific extensibility points of the Auth0 platform (like the Client Credentials grant). Auth0 invokes the Hooks at runtime to execute your custom logic.
```
```suggestion
Hooks allow you to customize the behavior of Auth0 using Node.js code. They are actually [Webtasks](https://webtask.io/), associated with specific extensibility points of the Auth0 platform (like the Client Credentials grant). Auth0 invokes the Hooks at runtime to execute your custom logic.
```

Commit:
Typo fix in articles/api-auth/tutorials/client-credentials/customize-with-hooks.md

jeffreylees commented on articles/applications/guides/enable-third-party-apps.md:
```diff
@@ -11,32 +10,37 @@ contentType:
-## Steps 
+    Third-party applications can only authenticate users from [connections](/connections) flagged as domain-level connections. Domain-level connections can be enabled for selected first-party applications while also being open to all third-party application users for authentication.
+    
+2. Update your application's login page (if you use [Lock](/libraries/lock/v11) in the [Hosted Login Page](/hosted-pages/login), you must also:
```
```suggestion
2. Update your application's login page. If you use [Lock](/libraries/lock/v11) in the [Universal Login Page](/universal-login/classic), you must also:
```

We don't say Hosted Pages anymore, and there was no close to this parenthetical

Commit:
Update articles/applications/guides/enable-third-party-apps.md

jeffreylees commented on articles/applications/index.md:
```diff
@@ -1,41 +1,52 @@
+
+## Manage client secrets
+
+You can [rotate an app's Client Secret](/dashboard/guides/applications/rotate-client-secret)  using the Auth0 Dashboard or the [Management API](/api/management/guides/applications/rotate-client-secret)
```
```suggestion
You can [rotate an app's Client Secret](/dashboard/guides/applications/rotate-client-secret) using the Auth0 Dashboard or the [Management API](/api/management/guides/applications/rotate-client-secret)
```

Commit:
Update articles/applications/index.md

jeffreylees commented on articles/applications/guides/enable-third-party-apps.md:
```diff
@@ -11,32 +10,37 @@ contentType:
-To enable third-party applications for your tenant, you will need to:
+  - Upgrade to Lock version 11 or later
+  - Set the `__useTenantInfo: config.isThirdPartyClient` flag when instantiating Lock
+  - *For [Private Cloud](/private-cloud) users only*: Set the [`configurationBaseUrl` option](https://github.com/auth0/lock#other-options) to `https://{config.auth0Domain}/` when instantiating Lock
```
```suggestion
  - *For [Private Cloud](/private-cloud) users only*: Set the [`configurationBaseUrl` option](https://auth0.com/docs/libraries/lock/v11/configuration#configurationbaseurl-string-) to `https://{config.auth0Domain}/` when instantiating Lock
```

Commit:
Update articles/applications/guides/enable-third-party-apps.md

jeffreylees commented on articles/dashboard/guides/connections/view-connections.md:
```diff
@@ -29,5 +29,5 @@ This guide will show you how to view all of the available and configured [connec
 - [Set Up Social Connections](/dashboard/guides/connections/set-up-connections-social)
 - [Identity Providers Supported by Auth0](/identityproviders)
-- [Multiple Tenants](/applications/concepts/multiple-tenants)
+- [Create Multiple Tenants](/dashboard/guides/tenants/create-multiple-tenants.md)
```
```suggestion
- [Create Multiple Tenants](/dashboard/guides/tenants/create-multiple-tenants)
```

Commit:
Fix broken link articles/dashboard/guides/connections/view-connections.md

jeffreylees commented on articles/dashboard/reference/views-api.md:
```diff
@@ -20,6 +20,6 @@ Available views for your API include:
 - **Scopes**: allows you to [define the scopes](/scopes/current/guides/define-scopes-using-dashboard) for your API by setting scope names and descriptions. For more details, see [API Scopes](/scopes/current/api-scopes).
 
-- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. (By default, this grant is **enabled* for [Regular Web Applications and Machine to Machine Applications](/applications/concepts/app-types-auth0). You can authorize any listed application to request Access Tokens for your API. Optionally, you can select a subset of the defined s ...
+- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. (By default, this grant is **enabled* for [Regular Web Applications and Machine to Machine Applications](/applications). You can authorize any listed application to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to limit an authori ...
```
```suggestion
- **Machine to Machine Applications**: lists all applications for which the **Client Credentials** grant is **enabled**. (By default, this grant is **enabled** for [Regular Web Applications and Machine to Machine Applications](/applications). You can authorize any listed application to request Access Tokens for your API. Optionally, you can select a subset of the defined scopes to limit an authorized application's access to your API. 
```

Commit:
Update articles/dashboard/reference/views-api.md

tamigoodall committed:
Minor fixes

Minor fixes

tamigoodall committed:
Redirects, sidebar, note

Redirects, sidebar, notes

tamigoodall committed:
Update index.md

Typo fix

tamigoodall committed:
Update index.md

Clarified language
